src/App.js

```diff
@@ -1,26 +1,129 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import React, { Component } from 'react'
+import hornData from './data.js';
+import Animal from './animal.js'
+import app from './App.css'
```

Comment:
It looks like you're declaring the app variable here for styling purposes and then not using it. You can either import the stylesheet without the variable assignment 'import './App.css', or you can assign the stylesheet to a variable that you then use in your className attributes...className={app.allHeaders}.